### ISSUE TITLE
Page has form containing data-default@ -> static method is being called at every request to any page

### ISSUE THREAD

**Issue by pamtbaau** [getgrav/grav-plugin-form#519]
When a page contains a form with a `data-default@: '\Grav\Plugin\MyPlugin::myStaticMethod'`, the method will be called on every page request. Even for pages that do not contain the form.

Is this as designed?

**Comment by mahagr** [comment]
Yeah, right now the forms get always initialized due to backward compatibility.

It's something I've wanted to improve for some time now.
